project-koku/koku
# [COST-2317] OCP on cloud sql improvements

Fixes [COST-2317](https://issues.redhat.com/browse/COST-2317)

Changes proposed in this PR:
* Removes window function calculations in favor of a count(*) in a CTE
* Removes unnecessary subquery
* Bugfix to not exclude volume data for AWS/Azure (was already fixed in GCP)

Testing Instructions:
1. Run standard load test customer data flow and check for OCP on AWS/Azure/GCP data.
2. Note that the cost will be higher compared to main thanks to the bugfix to not exclude volume data


[Smoke Tests](https://ci.ext.devshift.net/job/project-koku-koku-pr-check/3950/)
